includes/polling/ports.inc.php

```diff
@@ -111,8 +111,14 @@
 $ifmib_oids = array_merge($data_oids, $stat_oids);
```

Comment:
I was going to mention that!  Looks like it is slowly reverting to the previous declaration.